lib/pbench/server/api/resources/users_api.py

```diff
@@ -299,19 +300,43 @@ def __init__(self, logger, auth):
         self.logger = logger
         self.auth = auth
 
-    def get_valid_target_user(self, target_username: str) -> User:
+    def get_valid_target_user(
+        self, target_username: str, rest_call: str
+    ) -> Tuple["User", HTTPStatus, str]:
         """
         Helper function to determine whether the API call is permitted for the target username
         Right now it is only permited for an admin user and the target user itself
         :param target_username: str
-        :return: target User object/ None
+        :param rest_call: str
```

Comment:
Can we have a better name than `rest_call`? That's very unspecific. Are we talking "endpoint" like `/api/v1/users` or HTTP method like `PUT`, or both? It's an obvious question when reading this code, and the name provides no hint. For that matter, giving the name and type here is pointless, because they're both provided in the method signature. The only reason to list parameters in the docstring is to explain what they mean ... but even with that, a good name is best.

As a side note, it seems this is something we'd find useful beyond the scope of this single class.